src/main/kotlin/com/openlattice/indexing/BackgroundLinkingIndexingService.kt

```diff
@@ -155,22 +154,28 @@ class BackgroundLinkingIndexingService(
     }
 
     private fun mapCandidates(
-            candidates: List<Triple<UUID, OffsetDateTime, Set<UUID>>>
-    ): Pair<Map<UUID, Map<UUID, OffsetDateTime>>, Set<UUID>> {
+            candidates: List<Triple<List<String>, UUID, OffsetDateTime>>
```

Comment:
what are the values in this triple?

Reply:
The first is the list of entity data key pairs (which are split), then the linking id and the last is the last_write